jck/jck_target_folder_check/README.md

```diff
@@ -10,19 +10,22 @@ The jck_target_folder_check.sh script is used to cross-check if all second-level
 
 - Checkout `jck_target_folder_check.sh to your local machine from https://github.com/adoptium/aqa-tests/tree/master/jck
 - Manually download/unpack the SDK you want to use at {someLocation}
-- export TEST_JDK_HOME={someLocation}
+- export TEST_JDK_HOME={someLocation} (path to java/bin folder)
 - git clone https://github.com/adoptium/aqa-tests.git at {somerWorkspaceLocation}
 - cd /{somerWorkspaceLocation}/aqa-tests
 - ./get.sh //This will checkout TKG under /{somerWorkspaceLocation}/aqa-tests
 - Run jck_target_folder_check.sh {jck-repo-name} {somerWorkspaceLocation}
-
+```
+Sample input to run script
+```
+$ jck_target_folder_check.sh git@github.ibm.com:runtimes/JCK21-unzipped.git <LOCAL_AQA_TESTS_REPO>
 ```
 
 # What the script does 
 
 
 ```
-1. Git clone the the given JCK repo (if they do not already exist under given {somerWorkspaceLocation}. 
+1. Git clone the the given JCK repo (if they do not already exist under given {somerWorkspaceLocation}).
```

Comment:
Should be `{someWorkspaceLocation}`? I think `{workspaceLocation}` will be more professional. Please also update the other places as well.